src/utils/diff/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as jsdiff from 'diff';
+import debug from 'debug';
+import { isUndefined, isNull } from 'lodash';
+import { cleanUp, stringify } from './util';
+import { FAILED_TO_GENERATE_DETAILED_DIFF } from '../../notifications/information-message';
+
+const debugLogger = debug('testcafe:util:diff');
+
+function unifiedDiff (actual: string, expected: string): string {
+    const msg   = jsdiff.createPatch('string', actual, expected);
+    const lines = msg.split('\n').splice(5);
+
+    return lines
+        .map(cleanUp)
```

Comment:
the method is called `cleanUp`, but it returns the string with modified markup. It's hard to understand the purpose of the method from its name